src/main/java/hudson/matrix/AxisDescriptor.java

```diff
@@ -54,7 +54,7 @@ public boolean isInstantiable() {
     /**
      * Makes sure that the given name is good as a axis name.
      *
-     * Aside from {@link Jenkins#checkGoodName()} this disallows ',' and
+     * Aside from {@link Jenkins#checkGoodName} this disallows ',' and
```

Comment:
The original code was plain wrong: the method takes a `String` parameter. You can either specify the full list of parameter types, or (if there are no overloads) not specify the parameter types.